examples/aws_ec2/aws/nodes.tf

```diff
@@ -53,7 +53,7 @@ resource "aws_instance" "rke-node" {
     }
 
     inline = [
-      "curl releases.rancher.com/install-docker/1.12.sh | bash",
+      "curl https://releases.rancher.com/install-docker/18.09.sh | sh",
```

Comment:
May be it'd be a better scope to add docker install url as a variable. What do you think??

```
-      "curl https://releases.rancher.com/install-docker/18.09.sh | sh",
+      "curl ${var.docker_install_url} | sh",
```

Added on `examples/aws_ec2/aws/variables.tf`
```
+
+variable "docker_install_url" {
+  default = "https://releases.rancher.com/install-docker/18.09.sh"
+}
```

Reply:
Sounds good 👍 I will extract the variable...